tiny_gltf.cc

```diff
@@ -0,0 +1,2 @@
+#define TINYGLTF_IMPLEMENTATION
```

Comment:
Can we disable `TINYGLTF_IMPLEMENTATION` when we install the header? Otherwise if someone includes tinygltf and this macro it will be built again.